### ISSUE TITLE
BUG - Issue upgrading to 0.3.0

### ISSUE THREAD

**Issue by bmayer99** [fivetran/dbt_netsuite#27]
**Are you a current Fivetran customer?** 
Yes, Benoit @ Stedi Inc.

**Describe the bug**
just tried to upgrade to v 0.3.0. getting the following error
```
Compilation Error in model int_netsuite__transaction_lines_w_accounting_period (models/intermediate/int_netsuite__transaction_lines_w_accounting_period.sql)
  Required var 'transactions' not found in config
```
I could not see any transactions var mentioned in the docs

**Steps to reproduce**
1. change packages.yml to 0.3.0
2. run `dbt clean && dbt deps`
3. run `dbt run`

**Expected behavior**
no error. the package config should work out of the box.

**Project variables configuration**

```yml
copy dbt_project.yml here
```

**Package Version**

```yml
packages:

  - package: fivetran/netsuite
    version: 0.3.0
```

**Warehouse**

 
- [ ] BigQuery
- [ ] Redshift
- [x] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Please indicate the level of urgency** 
no urgency, I can work using previous versions

**Are you interested in contributing to this package?** 

- [ ] Yes, I can do this and open a PR for your review.
- [ ] Possibly, but I'm not quite sure how to do this. I'd be happy to do a live coding session with someone to get this fixed. 
- [x] No, I'd prefer if someone else fixed this.  I don't have the time and/or don't know what the root cause of the problem is.

**Comment by fivetran-joemarkiewicz** [comment]
Hi @bmayer99 thanks for opening this issue! I actually looked into real quick as the intention for this version is to be able to run out of the box and I believe we indeed found a quick bug fix in the packages `dbt_project.yml` incorrectly calling the variables for the package. I just pushed a commit to the branch `bug/netsuite-var-config` that fixed the error on my end. Would you be able to try the working branch as a package dependency and let me know if the issue persists. If it doesn't we can move forward with merging this bug fix.

You can use the below config in your `packages.yml` to test the working branch:
```yml
packages:
  - git: https://github.com/fivetran/dbt_netsuite.git
    revision: bug/netsuite-var-config
    warn-unpinned: false
```

Let me know how this bug fix works!

**Comment by fivetran-joemarkiewicz** [comment]
@bmayer99 After testing on my end I am sure this was the cause of the error you were seeing. I have just published a release with this bug fix for `v0.3.1` which should be live on the dbt hub in the morning EST. Let me know if you have any questions or issues on the new release. Thank you!

**Comment by bmayer99** [comment]
thank you @fivetran-joemarkiewicz 🙌🏼
